tests/ttnn/unit_tests/operations/test_distributed_layernorm.py

```diff
@@ -40,12 +40,12 @@ def tt_distributed_layernorm(inp, gamma, beta, epsilon, is_rmsnorm, compute_kern
                 )
             )
 
+    tt_stats = ttnn.aggregate_as_tensor(tt_stats)
```

Comment:
@SeanNijjar is this the name we want?  Our objective is to make our API familiar to PyTorch users.

Reply:
Hey @eyonland, which part of the expression are you referring to here? If the RHS, I am not sure. I'm not up to speed on what's idiomatic pytorch so I couldn't say if the name is appropriate. 

In c++ an aggregate is more like a struct so It doesn't _sound_ like what I'd expect at first but it looks like python frameworks like pandas use aggregate for collecting some dataset so it could apply. 

I don't have a valuable opinion on this.